test/webdev_proc_utils_test.dart

```diff
@@ -35,9 +37,20 @@ void main() {
       expect(getGlobalWebdevVersion(), isNull);
     });
 
-    test('with webdev activated', () async {
-      await activateWebdev('2.0.0');
-      expect(getGlobalWebdevVersion(), Version.parse('2.0.0'));
+    test('with webdev activated (dart 2)', () async {
+      print(Platform.version);
+      if (Platform.version.contains('2.19')) {
+        await activateWebdev('2.0.0');
+        expect(getGlobalWebdevVersion(), Version.parse('2.0.0'));
+      }
+    });
+
+    test('with webdev activated (dart 3)', () async {
+      print(Platform.version);
+      if (Platform.version.contains('3.')) {
```

Comment:
#nit, its a bit weird to have a unit test that doesn't actually test anything

Thoughts on merging both of these, and throwing if Platform.version is not supported?